src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,26 @@
+// #region media
+@mixin on-mobile {
+  @media (min-width: 488px) {
+    @content;
+  }
+}
+
+@mixin on-tabile {
```

Comment:
There is a typo in the mixin name `on-tabile`. It should likely be `on-tablet` to reflect the intended device size.